### DISCUSSION TITLE
Did "Check for update" and auto-update get removed?

### DISCUSSION THREAD

**Discussion by deviants0ul** [SnosMe/awakened-poe-trade#825] [Q&A]
I noticed those options were gone after the last update from my system tray icon.  Was this intentional?  First time commenting on Github, so hopefully this is the correct place to ask this.

Thanks for the awesome app!

**Comment by SnosMe** [comment]
App keeps auto-updating and checks for new one every 16 hours.
Menu item was visually removed, maybe I'll add it back later or in another place.
